website/static/js/contribAdder.js

```diff
@@ -337,7 +320,10 @@ var AddContributorViewModel = oop.extend(Paginator, {
         return $osf.postJSON(
             nodeApiUrl + 'contributors/', {
                 users: self.selection().map(function(user) {
-                    return ko.toJS(user);
+                    var permission = user.permission().value; //removes the value from the object
+                    var tUser = JSON.parse(ko.toJSON(user)); //The serialized user minus functions
```

Comment:
Oh, I should have read the comment. You needed to remove the methods. Disregard previous comment.